README.md

```diff
@@ -27,7 +27,7 @@ This means that the only thing you need to do is to add a one-liner to the scrip
 For bash, it suffices to add the following to .bashrc.
 
 ``` shell
-PS1+='$(hastory_gather_)'
+PROMPT_COMMAND="hastory_gather_"
```

Comment:
You also need to update the docs for `zsh`, that now needs something like this:
```
precmd() { hastory_gather_ }
```

Reply:
I changed the readme. With the zsh shell, the timestamps seem to be correct.